dev/com.ibm.ws.security.mp.jwt/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -86,3 +86,11 @@ cookieName.desc=The name of the cookie that is expected to contain a MicroProfil
 # Do not translate "Content Encryption Key", "JWE", "MP-JWT", "mp.jwt.decrypt.key.location", or "MicroProfile Config"
 keyManagementKeyAlias=Key management key alias
 keyManagementKeyAlias.desc=Private key alias of the key management key that is used to decrypt the Content Encryption Key. This private key must correspond to the public key that is used to encrypt the Content Encryption Key. This attribute is only used by versions 1.2 and beyond of the MP-JWT feature. This value overrides the mp.jwt.decrypt.key.location MicroProfile Config property if one is configured.
+
+# Do not translate "Content Encryption Key", "JWE", "MP-JWT", "mp.jwt.decrypt.key.algorithm", or "MicroProfile Config"
+keyManagementKeyAlgorithm=Key management key algorithm
+keyManagementKeyAlgorithm.desc=Specifies the decryption algorithm that is used to decrypt the Content Encryption Key. This attribute is only used by versions 2.1 and beyond of the MP-JWT feature. This value overrides the mp.jwt.decrypt.key.algorithm MicroProfile Config property if one is configured.
+
+# Do not translate "iat", "JWT", "MP-JWT", "mp.jwt.verify.token.age", or "MicroProfile Config"
+tokenAge=The time duration allowed since the JWT token was issued
+tokenAge.desc=Specifies the allowed token age in seconds when validating the JSON web token. This attribute is only used by versions 2.1 and beyond of the MP-JWT feature. The iat claim must be present in the JWT token and the configured number of seconds since iat must not have elapsed. If it has elapsed, then the request is rejected with an unauthorized response. This value overrides the mp.jwt.verify.token.age MicroProfile Config property if one is configured.
```

Comment:
Replace with the following text:
"...This attribute is used only in versions 2.1 and later of the MP-JWT feature. The issued at (iat) claim must be present in the JWT token. The configured number of seconds since iat must not have elapsed. If it has elapsed, then the request is rejected with an unauthorized response. The specified token age overrides the mp.jwt.verify.token.age MicroProfile Config property if it is configured.”